src/app.rs

```diff
@@ -432,7 +432,7 @@ pub async fn use_table(
     let target_table: Option<&String> = cx
         .overwritten_table_name
         .as_ref()
-        .or_else(|| positional_arg_table_name.as_ref());
+        .or(positional_arg_table_name.as_ref());
```

Comment:
This change raises the following clippy error.
```
$ cargo clippy -- -D warnings
    Checking dynein v0.2.0 (/<HOMEDIR>/dynein)
error: use of `or` followed by a function call
   --> src/app.rs:435:10
    |
435 |         .or(positional_arg_table_name.as_ref());
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `or_else(|| positional_arg_table_name.as_ref())`
    |
    = note: `-D clippy::or-fun-call` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#or_fun_call

error: could not compile `dynein` due to previous error
```

Could you revert this?

Reply:
I reverted it. You're seeing the error because dynein is still using Rust 1.56.

Happy to open another PR to upgrade to 1.59 once this PR is merged.